DOCUMENTATION.md

```diff
@@ -0,0 +1,116 @@
+# Documentation
+
+## Flatseal permissions
+
+This section indicates the permissions of Flatpak with better integration with Flatseal. It is based on the official [Sandbox Permissions Reference](https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html).
+
+Flatseal uses the `flatpak override` command. Every change the user does will make Flatseal run the `flatpak override` command to override the default permissions that the Flatpak program originally shipped with. All overridden permissions are located inside the `~/.local/share/flatpak/overrides` and `/var/lib/flatpak/overrides` directories. If you want to read more into `flatpak override`, you can look at the [`FLATPAK OVERRIDE(1)` manpage](https://manpages.debian.org/testing/flatpak/flatpak-override.1.en.html).
+
+### Permissions
+
+#### Share
+
+List of subsystems shared with the host system.
+
+Name | Permission(s) | Type | Description
+--- | --- | --- | ---
+Network | `--share=network` and `--unshare=network` | Toggle | Permit access to the network (`--share=network`); prohibit access to the network (`--unshare=network`).
```

Comment:
It definitely helps to have the command reference but, for the last column, we could simply explain what each state of the toggle means without referencing the command.

Reply:
The only reason I reference the commands is because there are people that would want to learn the commands of Flatpak without using a GUI (me for example).

I personally think that referencing the commands is a good idea because it helps the user learn the commands in case they want to learn the commands.

I don't mind removing, though. I just want to state my opinion.

Reply:
I think is fine to leave the references, I am just saying that, since this will be the documentation for Flatseal, it would be better to use address Flatseal options more directly, e.g. toggle on to allow access to the network, and toggle off to disallow access to the network.

Reply:
I see what you mean. Unfortunately bullet-points don't work in a tables when using Markdown. That would've made everything much easier.

Reply:
It looks better now!

Reply:
Feel free to resolve the conversation